### ISSUE TITLE
Spurious warnings about motion token uses

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/stylelint-plugin-carbon-tokens#127]
Given SCSS like:

```scss
@use "@carbon/motion" as motion;

.foo {
    transition: opacity motion.$duration-fast-02;
}
```

I get the spurious carbon/motion-duration-use error that:

> ✖  Expected carbon motion token, mixin or function to be set for variable "motion.$duration-fast-02" used by            
>            "transition" found "an unknown, undefined or unrecognized value".

(AFAICT I *am* using a carbon motion token.)

Possibly this is related to still being on Carbon 10.  We are in the process of upgrading to Carbon 11, but I haven't checked there yet.  I remember there being something weird about the number of dashes changing, like $duration--fast-02 --> $duration-fast-02, or vice-versa.

**Comment by lee-chase** [comment]
In Carbon 10 this should be `motion.$duration--fast-02`

**Comment by wkeese** [comment]
Agreed.  In any case, we've recentlly upgraded to Carbon 11, and I'm not getting spurious warnings anymore, for example this passes:

```
@use "@carbon/react/scss/motion" as motion;
...
    transition: width motion.$duration-moderate-02
      motion.motion(standard, expressive);
```

Not sure if Carbon 10 is working correctly or not but it doesn't seem important since everyone is moving to Carbon 11 now.

So I'm going to close this.
